static/admin/config.yml

```diff
@@ -32,3 +32,20 @@ collections:
               - { name: "subHeadline", widget: "string" }
               - { name: "metaTitle", widget: "string" }
               - { name: "content", widget: "markdown" }
+  - name: "faqEntry"
+    label: "FaqEntry"
+    folder: "src/data/faqEntry"
+    create: true
+    format: "json"
```

Comment:
Documentation for these options: https://www.netlifycms.org/docs/configuration-options/#collections